housing_modeling/src/create_dataset.py

```diff
@@ -0,0 +1,109 @@
+import os
+import tarfile
+from pathlib import Path
+
+import numpy as np
+import pandas as pd
+from six.moves import urllib
+from sklearn.model_selection import StratifiedShuffleSplit, train_test_split
+
+from .utils import create_logger
+
+path = Path(os.getcwd())
```

Comment:
this gives the path from where the script is run.. not the folder containing the current file. u can get the latter path using `os.path.dirname(os.path.abspath(__file__))`